docker/transfer-frontend/Dockerfile

```diff
@@ -6,8 +6,10 @@ RUN apt-get update && \
         curl -sS -o - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add && \
         echo "deb http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google-chrome.list && \
         echo "deb https://dl.bintray.com/sbt/debian /" >> /etc/apt/sources.list.d/sbt.list && \ 
+        echo "deb http://ppa.launchpad.net/ubuntu-mozilla-security/ppa/ubuntu bionic main" >> /etc/apt/sources.list.d/firefox.list && \ 
         apt-key adv --keyserver hkps://keyserver.ubuntu.com:443 --recv 2EE0EA64E40A89B84B2DF73499E82A75642AC823 && \    
+	    apt-key adv --keyserver keyserver.ubuntu.com --recv-keys A6DCF7707EBC211F && \
```

Comment:
Now it's indented too far. 🤔 Is your editor mixing up tabs and spaces?

Reply:
I'm sure it's vim. If I ever edit anything in vim then it messes up the spaces. There's probably a setting somewhere. Should be better now.